src/lib/pci_ahci.c

```diff
@@ -105,16 +105,14 @@ enum sata_fis_type {
  * Debug printf
  */
 #ifdef AHCI_DEBUG
-static FILE *dbg;
 #define DPRINTF(format, ...) \
 	do { \
-		fprintf(dbg, format, __VA_ARGS__); \
-		fflush(dbg); \
+		log_info(format, __VA_ARGS__); \
```

Comment:
there shouldn't be any need to change this debug code. this won't be enabled for release builds and there should be no need to redirect to a logger.

Reply:
I thought it would help the developers to have a consistent behavior.  I'll remove that.

Reply:
Well, it turns out that there is still a lot of "debug" messages (DPRINTF) that do show on D4D as of today.  They should be part of hyperkit's log like the others.  So I will make again a proposal for these debug logs.